pepper-apis/src/test/java/org/broadinstitute/ddp/route/SendEmailRouteTest.java

```diff
@@ -82,17 +82,18 @@ public static void tearDown() {
                     int numRows = handle.attach(QueuedEventDao.class)
                             .deleteQueuedEventsByEventConfigurationId(insertedKnownUserEventConfigId);
                     if (numRows != 1) {
-                        Assert.fail("Deleted " + numRows + " queued events for configuration id " + insertedKnownUserEventConfigId);
+                        // Note: tests might run in parallel and other tests might have deleted the event already.
+                        // So we log a message here instead of failing the test.
+                        LOG.error("Deleted " + numRows + " queued events for configuration id " + insertedKnownUserEventConfigId);
```

Comment:
Hopefully this will fix these:
![Screen Shot 2021-04-13 at 4 28 00 PM](https://user-images.githubusercontent.com/5713833/114617184-ce3ad180-9c75-11eb-8794-8cc176e25c6b.png)